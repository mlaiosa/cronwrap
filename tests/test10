#!/bin/sh

set -e

FILE=

if [ -e /tmp/$$ ];then
	echo /tmp/$$ exists, but I have pid $$.
	exit 1
fi

mkdir /tmp/$$
D=`pwd`
cd /tmp/$$

$D/cronwrap -- $D/tester a b --save-output=output.txt || true

if [ -e output.txt ]; then
	echo "-- didn't do its job."
fi

cd $D
rm -rf /tmp/$$

echo goodbye
