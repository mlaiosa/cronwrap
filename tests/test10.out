a
b
--save-output=output.txt
goodbye
